mage_ai/frontend/components/CodeEditor/index.tsx

```diff
@@ -163,6 +163,10 @@ function CodeEditor({
       setTextareaFocused?.(true);
     });
 
+    editor.onDidBlurEditorWidget(() => {
+      setTextareaFocused?.(false);
```

Comment:
`textareaFocused` was previously not accurate because we didn't set it to false when text area focus was lost